src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/Annotations/Anchoring/TreeNodeSelectionProcessor.cs

```diff
@@ -101,12 +101,12 @@ public override IList<DependencyObject> GetSelectedNodes(Object selection)
         public override UIElement GetParent(Object selection)
         {
             if (selection == null)
-                throw new ArgumentNullException("selection");
+                throw new ArgumentNullException(nameof(selection));
 
             UIElement element = selection as UIElement;
             if (element == null)
             {
-                throw new ArgumentException(SR.Get(SRID.WrongSelectionType), "selection");
+                throw new ArgumentException(SR.Get(SRID.WrongSelectionType), nameof(element));
```

Comment:
i think this makes sense to be`nameof(selection)` since that the `ArgumentException` is expressing that the `selection` paramter is the wrong type.